CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * [#268](https://github.com/dblock/waffle/pull/301): Cannot log in automatically on machine where Tomcat service is running
 * [#274](https://github.com/dblock/waffle/pull/274): Update WindowsSecurityContextImpl.java to handle SEC_E_BUFFER_TOO_SMALL
 * [#128](https://github.com/dblock/waffle/pull/128): Update WindowsSecurityContext.cs to handle SEC_E_BUFFER_TOO_SMALL
+* [#310](https://github.com/dblock/waffle/pull/310): Add equals and hashCode to WindowsPrincipal
```

Comment:
I can look into fixing it.  I'd love to automate it.  I'll look into that as well.  It's otherwise prone to human error ;)


Sent from Outlook Mobile






On Wed, Jan 13, 2016 at 4:08 AM -0800, "Daniel Doubrovkine (dB.) @dblockdotorg" <notifications@github.com> wrote:





> @@ -11,6 +11,7 @@
>  * [#268](https://github.com/dblock/waffle/pull/301): Cannot log in automatically on machine where Tomcat service is running
>  * [#274](https://github.com/dblock/waffle/pull/274): Update WindowsSecurityContextImpl.java to handle SEC_E_BUFFER_TOO_SMALL
>  * [#128](https://github.com/dblock/waffle/pull/128): Update WindowsSecurityContext.cs to handle SEC_E_BUFFER_TOO_SMALL
> +* [#310](https://github.com/dblock/waffle/pull/310): Add equals and hashCode to WindowsPrincipal

Sup with this changelog losing all the convention of having a period at the end and the name of the contributor? Maybe not in this PR, but someone please make  it look more consistent :)

---
Reply to this email directly or view it on GitHub:
https://github.com/dblock/waffle/pull/310/files#r49582512